wire-ios-sync-engine/Source/SessionManager/SessionFactories.swift

```diff
@@ -17,10 +17,36 @@
 //
 
 import avs
+import WireAPI
 import WireDataModel
 
+
+final class InMemoryAuthenticationStorage: AuthenticationStorage {
```

Comment:
I understand this is temporary too until renew access token PR is merged, right?
otherwise I would suggest to move to own file

Reply:
It is temporary until the renew access token PR is merged.

@johnxnguyen Do we want to make and keep it public? It seemed to intended for testing purposes.

Reply:
The `AuthenticationStorage` and it's conformers are now deleted, instead we can make use of `AuthenticationManager`.

Reply:
Updated to `AuthenticationManager`